File: src/main/java/com/itsci/mju/maebanjumpen/penalty/dto/PenaltyDTO.java
package com.itsci.mju.maebanjumpen.penalty.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class PenaltyDTO {
	private Integer penaltyId;
	private String penaltyType;
	private String penaltyDetail;
	private LocalDateTime penaltyDate;
	private String penaltyStatus;

	// ⬅️ เพิ่ม Report ID เพื่อใช้ในการเชื่อมโยง/อัปเดตตรรกะใน Service
	private Integer reportId;
}

File: src/main/kotlin/com/itsci/mju/maebanjumpen/penalty/dto/PenaltyDTO.kt
package com.itsci.mju.maebanjumpen.penalty.dto

import java.time.LocalDateTime

data class PenaltyDTO(
    var id: Long? = 0,
    var penaltyType: String? = null,
    var penaltyDetail: String? = null,
    var penaltyDate: LocalDateTime? = null,
    var penaltyStatus: String? = null,
    // เพิ่ม Report ID เพื่อใช้ในการเชื่อมโยง/อัปเดตตรรกะใน Service
    var reportId: Int? = null
)

